lib/json/ld/context.ex

```diff
@@ -46,7 +46,47 @@ defmodule JSON.LD.Context do
 
   # 3.2) If context is a string, [it's interpreted as a remote context]
   defp do_update(%JSON.LD.Context{} = active, local, remote, options) when is_binary(local) do
-    # TODO: fetch remote context and call recursively with remote updated
+    # 3.2.1)
+    local = absolute_iri(local, base(active))
+
+    # 3.2.2)
+    remote = case Enum.member?(remote, local) do
```

Comment:
Just a little bit cosmetics:

- There's no point of using `case` here, could you please use an `if` instead. 
- Can you use the `in` operator here, instead of `Enum.member?`.